ntp/server.sls

```diff
@@ -1,4 +1,5 @@
 {%- from "ntp/map.jinja" import server with context %}
 {%- if server.enabled %}
-
-{%- endif %}
\ No newline at end of file
+#include:
+#  - ntp.client
```

Comment:
Usually an client is always specified and client.sls is being call from init. Rasonings to keep include ntp.client in ntp.server are: 
- if directly ntp.server state is call (set ntp.server.enabled by default to true)
- an case where Hw server itself is NTP server without external peers.

Cons:
if client is specified, then we include it for 2nd time in server.sls will call it twice if just "ntp.init" is called.

So uncommenting it is no harm. I would vote for it.